Makefile

```diff
@@ -52,15 +53,17 @@ simplecross: $(BUILD_DIR) ## Cross-compile go binaries without using CGO.
 	env GOOS=linux GOARCH=amd64 go build -o $(BUILD_DIR)/linux-amd64-$(BIN_NAME) main.go
 	env GOOS=darwin GOARCH=amd64 go build -o $(BUILD_DIR)/darwin-amd64-$(BIN_NAME) main.go
 
-.PHONY: install
-install: build ## Install the go binary.
-	$(RM) $(INSTALL_DIR)/$(BIN_NAME)
-	cp $(BUILD_DIR)/$(BIN_NAME) $(INSTALL_DIR)
-
 .PHONY: clean
 clean: ## Clean the binary folder.
 	$(RM) -r $(BUILD_DIR)
 
+##@ Run
+
+.PHONY: run
+run: ## Run the go program.
+	go run main.go
```

Comment:
This is an odd run command, but I guess it would make sure the program actually executes.

Reply:
Yeah, I realised that it's not very useful, I'm going to remove it.